ckanext/switzerland/dcat-ap-switzerland_scheming.json

```diff
@@ -63,6 +63,18 @@
       },
       "preset": "dataset_organization"
     },
+    {
+      "field_name": "scheduled",
+      "label": {
+        "en": "Scheduled publication date on portal (set visibility to draft to publish it automatically)",
+        "de": "Geplanter Veröffentlichungstermin im Portal (Sichtbarkeit auf Entwurf setzen, um diesen Datensatz automatisch zu veröffentlichen)",
+        "fr": "Date de publication sur le portail (Définir la visibilité sur brouillon pour le publier automatiquement)",
+        "it": "Data di pubblicazione sul portale (Imposta la visibilità su bozza per pubblicare automaticamente il set di dati)"
+      },
+      "preset": "ogdch_date",
+      "validators": "scheming_required date_string_to_timestamp",
```

Comment:
These are the same validators as in the preset, so you should be able to leave them out of this config.

Reply:
Good point, I will remove that :)